src/components/Pages/ManageWiki/Cards/QuestyCaptcha.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <v-card>
+    <v-card-title>Additional Spam Protection</v-card-title>
+    <v-card-text class="pb-2">
+      QuestyCaptcha offers an extra layer of protection against spam accounts. During account creation, users will have to answer a question, which can be defined in settings. For more information on QuestyCaptcha, please visit the documentation page
+    </v-card-text>
+    <v-col class="checkbox">
+      <v-checkbox label="Activate spam protection" v-model="captchaActivate"></v-checkbox>
+    </v-col>
+<!--    Setting Panel-->
+    <v-expansion-panels v-model="panel">
```

Comment:
I think this is the reason nothing is working; is this `v-model` new? Right now its referencing a thing (called `panel` that's not there) see the error in the browser console:
```
[Vue warn]: Property or method "panel" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. See: https://vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.

found in

---> <QuestyCaptcha> at src/components/Pages/ManageWiki/Cards/QuestyCaptcha.vue
       <VMain>
         <ManageWiki> at src/components/Pages/ManageWiki/Tabs/ManageWiki.vue
           <VTabItem>
             <VTabsItems>
               <TabSettings> at src/components/Pages/ManageWiki/TabSettings.vue
                 <VApp>
                   <App> at src/App.vue
```

Reply:
I don't think so, it was added there to collapse the panel when saving is done (as Charlie said). 

https://github.com/wbstack/ui/pull/760/files#:~:text=%7D)-,this.panel%20%3D%20false,-%7D)